### DISCUSSION TITLE
new area: Forest

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#474] [Ideas]
this is related to #116 and some [discord discussion](https://discord.com/channels/714539345050075176/722219446973890580/1195438591275434005) that took place.

some properties of the forest:
- a purchasable area that expands the farm
- works similar to the field, but with different plot sizes (we're thinking 36x72)
- can grow trees in this area (such as apples, bananas, oranges, almonds, avocados, cherries, etc)
- after rainy days, mushrooms can spawn that can be foraged and used for recipes such as mushroom soup, a mushroom burger, and mushroom compost
- adds use for a new tool: axe

some details for trees:
- grown from saplings which can be purchased just like how seeds are purchased
- trees take a relatively long time to grow (relative to crops), but once grown they continuously produce fruits after fully grown
- can be chopped down using the axe to gather wood (higher level axes gather larger amounts of wood)
- fruits can be sold for profit (they fluctuate in price like other crops) and used for recipes (pies, jams, fermentations, etc)

what else?

**Comment by jeremyckahn** [comment]
This sounds great @lstebner! I jotted down some notes of my own, many of which parallel your own:

### Unlocking the Forest

1. Player purchases a Forest
  - We may consider Forest expansions similar to the Field, but that can be added post-MVP
  - The initial forest should be prohibitively expensive, as this is meant to be a late-game feature.
2. When purchased, the Forest screen is available between Cows and Workshop screens

### Forest mechanics

- The initial Forest has spots for 8 trees
- Tree seeds can be purchased from the shop. Tree seeds are unlocked via level ups similar to crop seeds (starting at level 44)
- Trees mature every day automatically
- Trees have long maturity timelines relative to crops. As a baseline, a tree might take 30 days to mature and start bearing fruit.
- Once mature, trees continually bear fruit
- Fruit maturation timelines are similar to crop maturation timelines
- There is a small chance (~10%) that storm will destroy a tree at any given time due to a lightning strike
- Trees can be chopped down any time after reaching maturity to be turned into wood (this can wait until post-MVP)

**Comment by jeremyckahn** [comment]
> after rainy days, mushrooms can spawn that can be foraged and used for recipes such as mushroom soup, a mushroom burger, and mushroom compost

I recommend deferring this idea until post-MVP.

**Comment by jeremyckahn** [comment]
@lstebner I think there's enough here to get started with in terms of the design of the feature. I imagine a lot of the functional details will be figured out in an exploratory manner. As we discussed [on Discord](https://discord.com/channels/714539345050075176/847884323330850847/1196639692964245514), I'll be here to bounce ideas off of and support development! 🙂
